software/firmware/bootloader.py

```diff
@@ -0,0 +1,56 @@
+from europi import *
+import utime as time
+from machine import reset
+
+# In order for a script to be included in the menu, it must be imported here
+from contrib.coin_toss import CoinToss
+from contrib.consequencer import Consequencer
+from contrib.diagnostic import Diagnostic
+from contrib.harmonic_lfos import HarmonicLFOs
+from contrib.radio_scanner import RadioScanner
+from contrib.polyrhythmic_sequencer import PolyrhythmSeq
+
+
+# Load all imported scripts that subclass EuroPiScript.
+SCRIPTS = [
+    CoinToss,
+    Consequencer,
+    Diagnostic,
+    HarmonicLFOs,
+    RadioScanner,
+    PolyrhythmSeq,
+]
+
+
+def display_choice(cls):
+    oled.centre_text(cls.__qualname__)
+
+
+def bootloader():
+    # Validate all scripts implement EuroPiScript or fail loudly.
+    for script in SCRIPTS:
+        if not issubclass(script, EuroPiScript):
+            raise NotImplementedError(f"Script does not implement EuroPiScript: {script}")
```

Comment:
I'm not sure if we should fail loud here, or silently omit non-EuroPiScript classes here. I really should revisit the EuroPiError class.